repository/spark/operator/params.yaml

```diff
@@ -114,6 +114,11 @@ parameters:
     default: ""
     displayName: "Spark History Server Persistent Volume Claim Name"
 
+  - name: awsCredentialsSecretName
```

Comment:
Is there another way of providing AWS credentials via secret without exposing it in the Operator configuration?